public/thrift/v1/index.html

```diff
@@ -8,8 +8,7 @@
       <link href="style.css" rel="stylesheet" type="text/css" />
       
       <title xmlns="">Zipkin V1 Thrift models</title>
-      
-   </head>
+      </head>
```

Comment:
changes to this file were generated as a part of me testing the script `generate_thrift_v1_docs/generate.sh`